modules/blogging/src/Volo.Blogging.Web/Pages/Blogs/Posts/Detail.cshtml

```diff
@@ -57,14 +57,14 @@
                                                 <i class="fa fa-comment"></i> @L["CommentWithCount", @Model.CommentCount]
 
 
-                                                @if (await Authorization.IsGrantedAsync(BloggingPermissions.Posts.Update))
+                                                @if (await Authorization.IsGrantedAsync(BloggingPermissions.Posts.Update) || (CurrentUser.Id.HasValue && CurrentUser.Id == Model.Post.CreatorId))
```

Comment:
`(CurrentUser.Id.HasValue && CurrentUser.Id == Model.Post.CreatorId)` can be `CurrentUser.Id == Model.Post.CreatorId`